tests/samples/INVENTORY.md

```diff
@@ -0,0 +1,13 @@
+# Sample spectra for becquerel
```

Comment:
For better PR records can this be moved to another PR?

Reply:
Sure, however it could make sense here because I am adding 3 new files to demonstrate and test the code in this PR (3 of a total of 9 files now).

Reply:
Totally, I think we should just add it quickly in another PR with the current files and then add the relevant lines in this PR. I'm interested in seeing in the milestone list so it is easier to track updates

Reply:
See #138